[DOCS] New exceptions UI/UX (#2546)
elastic/security-docs: docs/detections/detections-ui-exceptions.asciidoc

alerts are not generated. They provide a convenient way of allowing trusted
processes and network activity to function without producing unnecessary noise.

> You can add multiple exceptions to a single rule. An exception can also affect multiple rules.

Reviewer: ```suggestion
You can add multiple exceptions to a single rule. An exception can also apply to multiple rules.
```
if you think this makes sense

* To add an exception from the rule details page:
.. Go to the rule details page of the rule to which you want to add an
exception (*Manage* -> *Rules* -> *_<Rule name>_*).

> .. Scroll down the rule details page, select the *Rule Exceptions* tab, then click *Add rule exception*.

Reviewer 2: ```suggestion
.. Scroll down the rule details page, select the *Rule exceptions* tab, then click *Add rule exception*.
```

[role="screenshot"]
image::images/nested-exp.png[]

[float]
[[rules-using-same-exception]]
=== Find rules using the same exception

> From the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item and click *Affects X rules* to display the rules affected by the exception.

Reviewer 2: This section could use a little more context, maybe an extra sentence easing the reader into the idea of related/affected rules before jumping into actual instructions. Something like "If an exception is used by other rules, this is indicated on the exception" (not really an elegant sentence, but that general idea).

Author: How about:

```suggestion
An exception might be used by other rules. To find out, click on the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item, then click *Affects X rules*. 
```

Reviewer 3: This is good -- I like `select` vs. `click on`, especially when the user has a choice. I also think `X` is supposed to be in italics, (_x_), but that may not render with the bold already there -- can't remember. It may be OK. 

You could also flip this if you wanted: 

`"To find out if an exception is used by other rules...." `

Author: ```suggestion
To find out if an exception is used by other rules, select the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item, then click *Affects _X_ rules*. 
```

> From the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item and click *Affects X rules*. The rule or rules that the exception was added to displays.

Reviewer: ```suggestion
From the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item and click *Affects X rules* to display the rules affected by the exception.
```
